int8 agent_number
bool takeoff_command
